docs/formatcaps.rst

```diff
@@ -371,3 +371,45 @@ capabilities enabled in the chip and BIOS you will see:
     </guest>
 
   </capabilities>
+
+Host CPU model and features
+~~~~~~~~~~~~~~~~~
+
+As described in the (`Host capabilities`_) section, libvirt exposes to users the list of Host CPU features. Libvirt has a special way to expose this list: Instead of providing the full - and thereby often very long - set of features, libvirt specifies a CPU model name as baseline and additional features on top of it. 
+
+Example:
+
+::
+
+  <capabilities>
+
+    <host>
+      <uuid>7b55704c-29f4-11b2-a85c-9dc6ff50623f</uuid>
+      <cpu>
+        <arch>x86_64</arch>
+        <model>Skylake-Client-noTSX-IBRS</model>
+        <vendor>Intel</vendor>
+        ...
+        <feature name='ds'/>
+        <feature name='acpi'/>
+        <feature name='ss'/>
+        <feature name='ht'/>
+        <feature name='tm'/>
+...
+
+The ideal case would be that the baseline CPU model definition matches exactly the CPU present in the system and no additional feature is needed to express the capabilities of the CPU. For example, if you are running on a Server CPU you bought as ``Icelake`` type, the returned CPU model name could be ``Icelake-Server``. However, this ideal situation does rarely happen, mainly for two reasons:
+
+- Manufacturers do not ship one (=1) type of CPU under a given name, there are various different SKUs with different features under the same name. Yet it is not practical to have a database listing all of those variants. Instead Libvirt only has a list of a baseline CPU model names - roughly one per generation - in ``/usr/share/libvirt/cpu_map/``. It chooses the named type that shares the greatest number of features (CPUID bits and MSR features) with the actual CPU present in the machine and then lists the remaining named features as difference to that known name.
+
+- Some features might be in the hardware, but unavailable for various reasons (BIOS and kernel configuration, disabled for security, ...). One typical example where this situation happens is related to the TSX mitigation [1]. As a mitigation to the TAA side channel attack, the Linux kernel disables by default TSX and its 2 features, ``rtm`` and ``hle``. Since many Linux distributions keep this safer default behavior these 2 features appear as disabled.
+
+
+As a consequence, the list of detected features rarely match a named and defined CPU type.
+Sometimes that just means that you'll get the right name, but still a long list of features enabled or disabled on top of it.
+But in other times it might even lead to the situation that a different named CPU type, usually older CPU generation, is closer to the features libvirt finds in the CPU present in the system.
```

Comment:
```suggestion
But in other times it might even lead to the situation that a different named baseline model, usually an older CPU generation, is closer to the features libvirt finds in the CPU present in the system.
```